os/Kconfig

```diff
@@ -501,3 +501,11 @@ endmenu
 menu "Application Configuration"
 source "$APPSDIR/Kconfig"
 endmenu
+
+menu "Runtime Environment"
+source Kconfig.runtime
+endmenu
+
+menu "Device Management"
```

Comment:
Do you need to pick this out from framework?

Reply:
yes, we agreed it on the document